### PR TITLE
DietPi-Globals | G_RPI_UPDATE: Added APT hold for "libraspberrypi0", to hold complete "raspberrypi-firmware" package list

### PR DESCRIPTION
**Pull request by MichaIng**
Ref: https://github.com/Hexxeh/rpi-update, https://github.com/raspberrypi/firmware
- Should we let rpi-update remove old modules directories automatically or keep them in case of failure or unstable firmware?

Note:
- If one chooses to uninstall the firmware packages before (makes sense, removes docs and other unnecessary stuff, all 4 packages are well overwritten/replaced by rpi-update!), then `apt-mark hold ...` behaves a bid strange for not installed packages, leading to wrong/not intended hold states: https://bugs.launchpad.net/ubuntu/+source/apt/+bug/1694989
- But this works fine for installed packages, thus should be no issue for us and corrected as fast as APT packages are installed and hold state becomes relevant again.

### CODE REVIEW

**Comment by Fourdee** [comment]
@MichaIng 

> Should we let rpi-update remove old modules directories automatically or keep them in case of failure or unstable firmware?

We tried this in the past with patching our custom kernel (unzipped), and, ```dietpi-config```, which removed ```/lib/modules/*``` prior to run of ```rpi-update```. But some users had a failed update, rebooted and then missing modules.

Since then, we disabled the auto removal of current modules. As much as I love auto cleaning of old modules, its not worth the risk for end users in case of a failure.
PREP being the exclusion to this.

File: dietpi/func/dietpi-globals [hunk 1]
```diff
@@ -1416,10 +1416,8 @@ $print_logfile_info
 		if (( $G_HW_MODEL < 10 )); then
 
 			G_AG_CHECK_INSTALL_PREREQ rpi-update
-			export SKIP_BACKUP=1
-			export SKIP_WARNING=1
-			G_RUN_CMD rpi-update
-			apt-mark hold raspberrypi-bootloader raspberrypi-kernel libraspberrypi-bin
+			SKIP_BACKUP=1 SKIP_WARNING=1 G_RUN_CMD rpi-update #PRUNE_MODULES=1 would remove old/unused /lib/modules/* directories, too dangerous if update fails/firmware is unstable?
```
Comment (Fourdee) [inline dietpi/func/dietpi-globals:1419]: Nice 👍 

I did try this, but after ```G_RUN_CMD SKIP_BACKUP=1 SKIP_WARNING=1```, thats why it failed 😄

**Review by Fourdee — APPROVED** [review]
👍 Legend

File: dietpi/func/dietpi-globals (hunk 1 above)
Comment (MichaIng) [reply dietpi/func/dietpi-globals:1419]: @Fourdee 
Jep, also tried it the other way round first, then also was not sure if variables are forwarded through G_RUN_CMD to sub command, but it works 😃.

**Pull request merged**